code/datums/components/pellet_cloud.dm

```diff
@@ -189,10 +197,26 @@
 			break
 
 ///One of our pellets hit something, record what it was and check if we're done (terminated == num_pellets)
-/datum/component/pellet_cloud/proc/pellet_hit(obj/projectile/P, atom/movable/firer, atom/target, Angle)
+/datum/component/pellet_cloud/proc/pellet_hit(obj/projectile/P, atom/movable/firer, atom/target, Angle, hit_zone)
```

Comment:
leaving this here in case i dont see where you add it to the existing proc overwrites/calls as well.

Reply:
added it to the signals def